walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -64,22 +62,50 @@
     Collection<String> attendees = new ArrayList<>();
     attendees.addAll(request.getAttendees());
 
+    // Create a list of all attendees (optional and mandatory) for the requested meeting
+    Collection<String> mandatoryAndOptionalAttendees = new ArrayList<>();
+    mandatoryAndOptionalAttendees.addAll(request.getOptionalAttendees());
+    mandatoryAndOptionalAttendees.addAll(attendees);
+
     // Collection of timeranges that can not work for the proposed schedule
     // These are times when the attendees have an event scheduled at that particular time
     List<TimeRange> attendeesCannotScheduleHere = cannotSheduleMeeting(attendees, events); 
 
-    // Return the whole day if all attendees are free for the whole day
-    if (attendeesCannotScheduleHere.isEmpty()) 
+    // Get the timeranges for the optional attendees
```

Comment:
nit: update the comment to reflect that this is for mandatory + optional attendees